.eslintrc.js

```diff
@@ -15,7 +15,13 @@ module.exports = {
     jest: true
   },
   globals: {
-    ENV: true
+    ENV: true,
```

Comment:
I would propose we get rid of `ENV` while we are here. That smells like a project-specific hangover 👃

Reply:
Good idea 👍